streaming/README.md

```diff
@@ -0,0 +1,27 @@
+# Cosmos-SDK Plugins
```

Comment:
please don't put this in the root, we moved it to store because we are trying to house all storage related items in one place

Reply:
it now resides in `store/streaming`